src/main/java/com/questhelper/panel/QuestOverviewPanel.java

```diff
@@ -607,11 +604,11 @@ private void updateQuestNotes(List<String> notes)
 			for (String note : notes)
 			{
 				textNote.append(note);
-				textNote.append("<br><br>");
+				textNote.append("\n\n");
```

Comment:
Unrelated to this PR, but while we're here:
Can we use a boolean to ensure we only add newlines if there are more notes?
Like this:
```java
var first = true;
for (var note : notes) {
  if (!first) {
    textNote.append("\n\n");
  }
  textNote.append(note);
  first = false;
}
```